### PR TITLE
Tokens doc

### PR DESCRIPTION
**Pull request by kimia84**
## What does this PR do?
-issue #956-
Some more information on what tokens are and also different types of them. ~Docs~

## ✅ Pull Request checklist

- [x] Is this code complete?
- [ ] Are tests done/modified?
- [x] Are docs written for this PR? (if applicable)

## 🚨 Is this a breaking change for API clients?
No

## :squirrel: Deploy notes
N/A

### CODE REVIEW

File: frontend/src/test_file.py [hunk 1]
```diff
@@ -0,0 +1 @@
+fggefwqk    ;al
```
Comment (zipy124) [inline frontend/src/test_file.py]: @kimia84 happy to approve everything, whats this though?

File: frontend/src/components/documentation/Routes/OAuth/Token.jsx [hunk 2]
```diff
@@ -59,6 +59,48 @@ export default class Token extends React.Component {
...
+            <p>Note that you can also use OAuth Tokens to access all the same data that generic app tokens can access.
+            </p>
+
+            <p>Each token is uniquely generated for each user logging into each app.
+            </p>
+
+            <p>Please note, access to any of this data needs to be approved by the user first.
+            </p>
+
+            <p>To use this type of token for your app, you need include a “Sign in With UCL Button” in your app, such as the one provided below, which links users to the authorisation endpoint with your app’s Client ID (accessible via the dashboard) and a random state number included in the GET parameters.
```
Comment (zipy124) [inline frontend/src/components/documentation/Routes/OAuth/Token.jsx]: The sign in button isn't mandatory only a link to the endpoint. Maybe make clear its optional here?

File: frontend/src/test_file.py (hunk 1 above)
Comment (kimia84) [reply frontend/src/test_file.py]: It seems like the file was pushed by mistake - I have deleted it

File: frontend/src/components/documentation/Routes/OAuth/Token.jsx (hunk 2 above)
Comment (kimia84) [reply frontend/src/components/documentation/Routes/OAuth/Token.jsx]: Changed that line - Please let me know what you think now?

Comment (zipy124) [reply frontend/src/components/documentation/Routes/OAuth/Token.jsx]: All good now!

**Pull request merged**